draft-ietf-taps-interface.md

```diff
@@ -3341,7 +3341,9 @@ directly by the application or indirectly configured for a traffic profile.
 Applications should be aware that communication attempts can lead to more than one connection establishment.
 This is the case, for example, when the Transport Services system also executes name resolution, when support mechanisms such as
 TURN or ICE are used to establish connectivity, if protocols or paths are raised, or if a path fails and
-fallback or re-establishment is supported in the Transport Services system.
+fallback or re-establishment is supported in the Transport Services system. Applications should take special
+care when using 0-RTT session resumption (see {{prop-0rtt}), as early data sent across multiple paths during
+connection establishment may reveal information that can be used to correlate endpoints on multiple paths.
```

Comment:
Very strange, I can't find the button that lets me propose a text change anymore?  and I'm not sure how to code this directly, surely that could be done if one knew the right markdown tags....

Anyway: I propose to finish the sentence with "... used to correlate endpoints on these paths." instead of repeating "multiple". Just a suggestion.

Reply:
Good suggestion, incorporated. thanks!